### PR TITLE
Create production Storybook

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Compiles Storybook and creates an Nginx-based container.

Depends on https://github.com/libero/storybook/pull/3.

### CODE REVIEW

File: .docker/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    gzip_proxied any;
+    gzip_vary on;
+    gzip_types
+      application/javascript
+      image/svg+xml
+      text/css
+      text/json
+    ;
+
+    location ~* ^(/.+\.bundle\..+)|(/static/media/.+)$ {
```
Comment (thewilkybarkid) [inline .docker/nginx.conf:8]: Would be useful to know if there's a way to isolate Webpack-created files (ie with a version in the name); looks like Storybook doesn't let you touch this part of the Webpack config.

Comment (stephenwf) [reply .docker/nginx.conf:8]: There is a manifest plugin that will report out the assets created by Webpack, that could be used?

Comment (thewilkybarkid) [reply .docker/nginx.conf:8]: Sounds tricky to make use of. Ideally would just move them all to a specific subdirectory...

File: .docker/nginx.conf [hunk 2]
```diff
@@ -0,0 +1,38 @@
+server {
+    listen 8080;
+    server_name localhost;
+    root /usr/share/nginx/html;
+
+    gzip on;
```
Comment (giorgiosironi) [inline .docker/nginx.conf]: isn't that a bit out of scope for a configuration built into a Docker image? The same way we don't implement HTTPS in this context but we terminate it in a load balancer or proxy

**Review by giorgiosironi — APPROVED** [review]

File: .docker/nginx.conf (hunk 2 above)
Comment (thewilkybarkid) [reply .docker/nginx.conf]: Replaced with an `include` in https://github.com/libero/storybook/pull/4/commits/e75805749fc4a1751430e6e1c5eb8345053d7303.

**Review by giorgiosironi — APPROVED** [review]

**Comment by davidcmoulton** [comment]
Container isn't stopped by `ENV=prod make stop`.

**Comment by thewilkybarkid** [comment]
@davidcmoulton Not able to replicate...

**Pull request merged**